### PR TITLE
fixed #23795: UI Card Extension CSS Broken for Default Icons

### PR DESCRIPTION
**Pull request by alex40724**
Suggested fix for 23795: UI Card Extension CSS Broken for Default Icons
https://mantis.ilias.de/view.php?id=23795

### CODE REVIEW

**Comment by alex40724** [comment]
Stop! I need to fix some tests...

**Comment by Amstutz** [comment]
Was just typing a message, requesting exactly that ;-)

**Comment by alex40724** [comment]
done

File: src/UI/Component/Icon/Icon.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	const SMALL = 'small';
 	const MEDIUM = 'medium';
 	const LARGE = 'large';
-	const RESPONSIVE = 'responsive';
+	// const RESPONSIVE = 'responsive';		// currently not supported
```
Comment (Amstutz) [inline src/UI/Component/Icon/Icon.php]: Here you revert the public interface of icons. Leave that as is.

**Review by Amstutz — CHANGES_REQUESTED** [review]
Hi @alex40724 thx for the fixes. Since we currently have a broken component in the trunk, I would like to fix this ASAP, and therefore merge this today. However, in your PR is one line that changes the public interface of Icons (see my comment). For this we need the JF. Take this one line out, so we can merge this know. 

Note that we need to discuss the related bug in the next JF to check back, if all is good with ignoring the requirement of having evenly spread distribution of the section in the top section.

File: src/UI/Component/Icon/Icon.php (hunk 1 above)
Comment (alex40724) [reply src/UI/Component/Icon/Icon.php]: ok, done

**Comment by klees** [comment]
@Amstutz We really need to find some better approach for the tests of the rendering.

**Comment by Amstutz** [comment]
@alex40724 Thanks for the fix.

**Pull request merged**